README.md

```diff
@@ -5,27 +5,54 @@ This repository provides a template for creating **feature specifications** for
 
 When a new feature or product is to be developed, a new pull request is created, and contributors are invited to discuss its content and propose suggestions. The goal of a feature specification is to define the expected behavior on a high level and point out corner cases that need to be addressed.
 
+The person in charge of the PR (the owner) is the person assigned to the PR. This allows for changing ownership. When the PR assignee changes, both new and old assignees should be notified.
+
+#### Start
+
+To start a new specification, it is recommended to write the first draft by filling in only the `Summary` and `Motivation` parts. 
+
+The name of the spec must follow the pattern: `PR_number-feature-name.md`. For example, if PR number 12 is about facetting, its spec will be named `0012-facetting.md` or `0012-facetted-search.md`. It will be logical that the names of the PRs will not follow each other. This is not a problem. The fact that the name has the number of PR simplifies the fact that there is no synchronization to choose this number. 
+
+#### Draft PR
+
+Once the file is created with the two parts, `Summary` and `Motivation` filled in. It is time to create a draft PR. The PR should remain in draft until the content is, according to the author, ready for comments. However, outsiders can make some recommendations, but these are more in the spirit of helping rather than suggesting changes. 
+
+#### Open PR
+
+Once the specification is finished in the owner's eyes, he can then switch the PR to open. At that time, it can be commented on, modified, and challenged by these peers. If there is the slightest friction during this process, a discussion will be recommended offline within MeiliSearch. The PR owner will organize this discussion, and at the end of this interview, the owner will clearly describe the report of the meeting in the PR and make any necessary changes.
```

Comment:
What do you mean by `Once the specification is finished`? Because in the previous step you ask to fill the `Summary` and `Motivation` parts, but you don't mention the other parts. Should they be all completed when we pass the PR from `draft` to `open`? Can we remove the `draft` status if many sections are empty?

Reply:
Still have the same concerned 😅 -> not sure to understand `Once the specification is finished in the owner's eyes` means